tests/Tests/Arithmetization/Test4.hs

```diff
@@ -41,13 +43,16 @@ testZKP x ps targetValue =
 
         (omega, k1, k2) = getParams 5
         inputs  = fromList [(1, targetValue), (V.item $ acOutput ac, 1)]
-        plonk   = Plonk @32 omega k1 k2 (Vector @2 $ keys inputs) ac x
-        setupP  = setupProve @PlonkBS plonk
-        setupV  = setupVerify @PlonkBS plonk
+        plonk   = Plonk @32 omega k1 k2 (acOutput ac) ac x
+        setupP  = setupProve @(PlonkBS N) plonk
+        setupV  = setupVerify @(PlonkBS N) plonk
         witness = (PlonkWitnessInput inputs, ps)
-        (input, proof) = prove @PlonkBS setupP witness
+        (_, proof) = prove @(PlonkBS N) setupP witness
+
+        -- `one` corresponds to `True`
+        output = plonkVerifierInput $ V.singleton one
 
-    in verify @PlonkBS setupV input proof
+    in verify @(PlonkBS N) setupV output proof
```

Comment:
Maybe we should name it `none`?

Reply:
Do you mean `output`? We want to verify that the circuit returns symbolic `True` represented by `Zp 1`.

Reply:
There is a difference between inputs to the circuit and inputs to the protocol. The latter actually includes circuit outputs.

Reply:
I think something is wrong with `output` in input, maybe `noneInput` or `onlyOutput` ?

Reply:
Change to `circuitOuputsTrue`.